default/python/AI/FreeOrionAI.py

```diff
@@ -172,7 +172,7 @@ def handleChatMessage(sender_id, message_text):  # pylint: disable=invalid-name
     message_text is the text of the sent message."""
     empire = fo.getEmpire()
     if empire is None:
-        print "This client has no empire. Doing nothing to handle chat message."
+        info("This client has no empire. Doing nothing to handle chat message.")
```

Comment:
As above, this seems like enough of an unexpected situation that it should be debug or warn rather than merely info.